PostGreKeyRotation/PostGreKeyRotation/Models/PostgreSQLConnectResult.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections.Generic;
+
+namespace PostGreKeyRotation.Models
+{
+    public class PostgreSQLConnectResult
```

Comment:
Not to nitpick but perhaps this file should be my commit?

Reply:
Fixed